Modules/Loadable/Transforms/MRMLDM/vtkMRMLTransformsDisplayableManager3D.cxx

```diff
@@ -47,6 +53,9 @@
 #include <vtkRenderWindowInteractor.h>
 #include <vtkRenderer.h>
 #include <vtkSmartPointer.h>
+#include <vtkTransform.h>
+
+const double DEFAULT_SCALE[3] = {100.0, 100.0, 100.0};
```

Comment:
Usually scale is in the 0..1 range. Either rename to DEFAULT_SCALE_PERCENT (if the range is 0..100) or change the range to 0..1.